runtime/lua/vim/lsp/rpc.lua

```diff
@@ -608,6 +608,73 @@ function M.connect(host, port)
   end
 end
 
+---@class Dispatchers
```

Comment:
Since these types are globally scoped, we should "namespace" them (e.g. `vim.rpc.Dispatchers`).